addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -24,6 +23,12 @@ def setUpClass(cls):
             name='Marie Marketing',
             signature='--\nMarie'
         )
+        records = cls.records
+        mobile_numbers = [f'045300{x}{x}99' for x in range(6)] + ['0453000099'] * 4
+        _, new_records = cls._get_sms_test_records(mobile_numbers=mobile_numbers)
+        records += new_records
+        cls.records = cls._reset_mail_context(records)
```

Comment:
you don't need a variable anymore
either of of them, depending if one test fail or not
```py
cls.records += cls._get_sms_test_records(mobile_numbers=mobile_numbers)
cls.records += cls._reset_mail_context(cls._get_sms_test_records(mobile_numbers=mobile_numbers))
```